index.html

```diff
@@ -7,58 +7,268 @@
     <link rel="stylesheet" href="Styles.css" />
     <title>Portfolio Website:Anthony</title>
```

Comment:
- [ ] Please fix this weird space behind the headline section.
![image](https://user-images.githubusercontent.com/74744787/227737273-e15d0a66-1653-415a-a996-f8aa4e7d2f33.png)